CONTRIBUTING.rst

```diff
@@ -47,13 +48,13 @@ For other questions, discussions, and chats, we have:
   for scancode and other related tools. You can use your
   favorite IRC client or use the web chat at https://webchat.freenode.net/
 
-- a mailing list at https://lists.sourceforge.net/lists/listinfo/aboutcode-discuss
```

Comment:
I think here we should promote gitter and eventually remove references to the other ways.... that's too many other wise. What do you think?

Reply:
Correct because most of the things happen on Gitter, and not seen anyone on sourceforge subscribe list from last couple of days